packages/components/src/ObjectViewer/JSONLike/JSONLike.scss

```diff
@@ -91,6 +91,13 @@
 		color: $dove-gray;
 	}
 
+	.native-value-table {
```

Comment:
This is used in Table component, but it means that this component has to know the implementation of JSONLike component.
IMO 
* this should be defined in Table.scss
* JSONLike takes a native value className and doesn't resolve it from the theme, but set it directly
* Table component resolves from its theme and pass it to JSONLike